### PR TITLE
Update to Avalon core

### PR DESCRIPTION
**Pull request by BigRoy**

### CODE REVIEW

**Comment by BigRoy** [comment]
We need to test this on our pipeline to see if everything works. Specifically also check the work files app (added in Maya Avalon menu) to see if it doesn't allow users to break anything.

@aardschok can you have a look?

**Comment by aardschok** [comment]
The Work Files app works pretty well but it can create an issue with our `Version Up` script. The findings might be interesting for @tokejepsen 

In the Work Files app the user can add a comment which is added at the end of the file name, e.g.: `hero_model_v001_hi_res`, here `hi_res` is the "comment". The comment is not being checked if it contains anything which resembles versioning like `hero_model_v001_v001`

If we have a file open which is named `hero_model_v001_v001` the next version, logically is `hero_model_v001_v002` but we might want to go to the new "major" version `hero_model_v002`. The Work File app support this by leaving the comment field blank.

The Work Files app checks the new possible version based on the template which allows the user to fill any gabs. This logic is default the app and there is a downside to this

If we have 3 files: 
> `hero_model_v001, hero_model_v002, hero_model_v009`

The next version in Work Files app will be `hero_model_003` in stead of `hero_model_010` which is the next logical version as `v009` is the highest version available. The user will have to toggle of a checkbox and manually enter in version number.

In short:
Pros:
* Easy UI
* Adjustable template from project config
* Comments are optional

Cons:
* Comments are not checked for versioning
* Tool does not take current file in consideration (auto fill comments if it has any)
* Fills gabs, automatically picks the lowest next version

I think this tool can work for us if we step away from our `Version Up` script, what are you thought on this @BigRoy

**Comment by tokejepsen** [comment]
Very interesting findings and thanks for testing!

> Fills gabs, automatically picks the lowest next version

Sorry about the gab filling, should have picked up on this. Adjusting this to not fill gabs might be preferable as default.

>  but we might want to go to the new "major"

I did not consider major/minor versioning workflows. Could certainly look at adding that to the dataset for the template and input.
Might need some more info on this workflow, like what is the artist expected to change etc.
